test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -62,6 +62,16 @@ function RunTestCase
     }
 }
 
+Describe "Validate that <pshome>/<culture>/default.help.txt is present" -Tags @('CI') {
+
+    It "Get-Help Get-Help returns information about the help system." {
+
+        $helpContent = Get-Help Get-Help | Out-String
+        $helpContent | Should Match Get-Help
+        $helpContent | Should Match Update-Help
```

Comment:
>$helpContent | Should Match Update-Help [](start = 8, length = 39)

Do you also want to check that the file is present at the expected location under $PSHome?